server-config.json

```diff
@@ -1,14 +1,14 @@
 {
-    "url": "http://localhost:8090",
-    "port": 8090,
-    "relayHubAddress": "0x3bA95e1cccd397b5124BcdCC5bf0952114E6A701",
-    "relayVerifierAddress": "0x74Dc4471FA8C8fBE09c7a0C400a0852b0A9d04b2",
-    "deployVerifierAddress": "0x1938517B0762103d52590Ca21d459968c25c9E67",
-    "gasPriceFactor": 1,
-    "rskNodeUrl": "http://rskj:4444",
-    "devMode": true,
-    "customReplenish": false,
-    "allowForSponsoredTx": true,
-    "logLevel": 1,
-    "workdir": "/home/user/workspace/enveloping/environment"
+  "url": "http://localhost:8090",
+  "port": 8090,
+  "relayHubAddress": "0x3bA95e1cccd397b5124BcdCC5bf0952114E6A701",
+  "relayVerifierAddress": "0x74Dc4471FA8C8fBE09c7a0C400a0852b0A9d04b2",
+  "deployVerifierAddress": "0x1938517B0762103d52590Ca21d459968c25c9E67",
+  "gasPriceFactor": 1,
+  "rskNodeUrl": "http://rskj:4444",
+  "devMode": true,
+  "customReplenish": false,
+  "workerFeePercentage": "0",
```

Comment:
I would call it just `feePercentage`. Not necessary the fee will be collected by the worker.

Reply:
thought that was a bit ambiguous

Reply:
done